app/views/posts/_link_form.html.erb

```diff
@@ -22,7 +22,7 @@
   </div>
   <div class="create-post m-3">
     <div class="form-group">
-      <%= render 'checkbox_form' %>
+      <%= render 'content_warning_fields' %>
     </div>
   </div>
   <div>
```

Comment:
these lines are duplicated in 2 files now right? can we refactor to make it DRY?